### PR TITLE
fix(cli): quantity rounding

### LINKED ISSUE #1015: can't place order using CLI
```
xud@xud-test-2:~$ xucli buy 2.3 ltc/btc 0.0079

/opt/xud/node_modules/yargs/yargs.js:1101
      else throw err
           ^
AssertionError: Assertion failed
    at new goog.asserts.AssertionError (/opt/xud/node_modules/google-protobuf/google-protobuf.js:98:603)
    at Object.goog.asserts.doAssertFailure_ (/opt/xud/node_modules/google-protobuf/google-protobuf.js:99:126)
    at Object.goog.asserts.assert (/opt/xud/node_modules/google-protobuf/google-protobuf.js:99:385)
    at jspb.BinaryEncoder.writeUnsignedVarint64 (/opt/xud/node_modules/google-protobuf/google-protobuf.js:253:77)
    at jspb.BinaryWriter.writeUnsignedVarint64_ (/opt/xud/node_modules/google-protobuf/google-protobuf.js:273:507)
    at jspb.BinaryWriter.writeUint64 (/opt/xud/node_modules/google-protobuf/google-protobuf.js:278:318)
    at Function.proto.xudrpc.PlaceOrderRequest.serializeBinaryToWriter (/opt/xud/dist/proto/xudrpc_pb.js:6681:12)
    at proto.xudrpc.PlaceOrderRequest.serializeBinary (/opt/xud/dist/proto/xudrpc_pb.js:6658:34)
    at serialize_xudrpc_PlaceOrderRequest (/opt/xud/dist/proto/xudrpc_grpc_pb.js:330:26)
    at Object.final_requester.sendMessage (/opt/xud/node_modules/grpc/src/client_interceptors.js:806:37)
```

### PR DESCRIPTION
**Pull request by sangaman**
This fixes edge case rounding errors when converting cli arguments like `2.3 * 100000000 = 229999999.99999997`. It creates conversion methods between coins and satoshis as well as unit tests for those methods.

Fixes #1015.

### CODE REVIEW

File: test/unit/Command.spec.ts [hunk 1]
```diff
@@ -107,3 +108,21 @@ describe('Command.listorders.formatOrders', () => {
     expect(output[0].orders.length).to.equal(5);
   });
 });
+
+describe('Satoshi/Coin conversion', () => {
```
Comment (erkarl) [inline test/unit/Command.spec.ts]: Very happy to see these unit tests :).
